### ISSUE TITLE
Is there any way to connecting video stream without textureview preview?

### ISSUE THREAD

**Issue by Shaooooo** [pedroSG94/RTSP-Server#135]
I want to use VLC connect to my android app, and don't preview the camera screen in app. Can I?

**Comment by pedroSG94** [comment]
Yes, you can do it. You can use RtspServerStream class and never call startPreview or use your current class without the preview class in the constructor. 
If you are using the app example, you can do it like this:
```kotlin
rtspServerCamera1 = RtspServerCamera1(context, this, 1935)
```
